deep-security-agent/libraries/dsa.rb

```diff
@@ -108,7 +108,7 @@ def is_DSA_activated?
 
     if certHash == nil || certHash == ''
       raise "Could not query Deep Security Agent certificate. Please make sure Deep Security Agent has been installed."
-    elsif !BOOTSTRAP_CERT.casecmp(certHash)
+    elsif BOOTSTRAP_CERT.casecmp(certHash) != 0
```

Comment:
(Optional)
This could also be a boolean: 
https://ruby-doc.org/core-2.4.0/String.html#method-i-casecmp-3F

So this whole block could be:
```
if certHash == nil || certHash == ''
  raise "Could not query Deep Security Agent certificate. Please make sure Deep Security Agent has been installed."
end

activated = BOOTSTRAP_CERT.casecmp?(certHash) 
```